web-server/v0.4/components/Comparison/CompareResults.js

```diff
@@ -0,0 +1,469 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Select, Card, Spin, Tag, Table, Button } from 'antd';
+import history from '../../core/history';
+import axios from 'axios';
+import cloneDeep from 'lodash/cloneDeep';
+
+class CompareResults extends React.Component {
+  static propTypes = {
+    results: PropTypes.array,
+    controller: PropTypes.string
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      responseData: [],
+      responseDataAll: [],
+      loading: true,
+      loadingButton: false,
+      tables: [],
+      selectedRowKeys: [],
+      rowSelections: [],
+      selectedPort: "all",
+      ports: [],
+      configData: [],
+      selectedConfig: []
+    };
+  }
+
+  componentDidMount() {
+    this.setState({loading: true});
+    const { results } = this.props;
+    var iterationRequests = [];
+    for (var result in results) {
+      if (results[result].controller.includes('.')) {
+        iterationRequests.push(axios.get('http://pbench.perf.lab.eng.bos.redhat.com/results/' + encodeURI((results[result].controller).slice(0, (results[result].controller).indexOf("."))) +'/'+ encodeURI(results[result].result) + '/result.json'))
+      } else {
+        iterationRequests.push(axios.get('http://pbench.perf.lab.eng.bos.redhat.com/results/' + encodeURI((results[result].controller)) +'/'+ encodeURI(results[result].result) + '/result.json'))
```

Comment:
Can we find a way to remove this hard-coded `http://pbench.perf.lab.eng.bos.redhat.com` URL prefix?